data/json/items/comestibles/soup.json

```diff
@@ -235,6 +235,28 @@
     "vitamins": [ [ "vitC", 9 ], [ "iron", 3 ] ],
     "fun": 1
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "hellfire_stew",
+    "looks_like": "soup_woods",
+    "name": { "str": "hellfire stew" },
+    "weight": "250 g",
+    "color": "brown",
+    "spoils_in": "5 days",
+    "container": "can_medium",
+    "comestible_type": "DRINK",
+    "symbol": "o",
+    "quench": 2,
+    "calories": 504,
+    "description": "An old recipe from the American Civil War.  It's edible.",
```

Comment:
That is very little description For something with hellfire in the name.

Reply:
That it is.  I'll fix that.